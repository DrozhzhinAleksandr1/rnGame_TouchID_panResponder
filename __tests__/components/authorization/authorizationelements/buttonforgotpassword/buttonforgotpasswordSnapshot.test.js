import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import {ButtonForgotPassword}  from '../../../../../app/components/authorization/authoriztionelements/buttonforgotpassword';

it('create snapshot Main showMenu={true}', () => {
    const renderer = new ShallowRenderer()
    let buttonForgotPasswordSnapshot = renderer.render(<ButtonForgotPassword />);
    expect(buttonForgotPasswordSnapshot).toMatchSnapshot();
});