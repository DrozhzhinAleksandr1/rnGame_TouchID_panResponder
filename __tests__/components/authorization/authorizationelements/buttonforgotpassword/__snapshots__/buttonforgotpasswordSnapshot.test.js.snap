// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot Main showMenu={true} 1`] = `
<TouchableWithoutFeedback
  accessibilityLabel="ForgotPasswordAuthorizationID"
  onPress={[Function]}
  testID="ForgotPasswordAuthorizationID"
>
  <Component
    style={
      Object {
        "marginBottom": 50,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#FFFFFF",
          "fontFamily": "Roboto-Regular",
          "fontSize": 18,
        }
      }
    />
  </Component>
</TouchableWithoutFeedback>
`;
